src/Ticket.php

```diff
@@ -446,7 +446,7 @@ public function getDatasToAddSLA($slas_id, $entities_id, $date, $type)
             }
            // Compute time_to_resolve
             $data['sla_waiting_duration'] = $this->fields['sla_waiting_duration'] ?? 0;
```

Comment:
```suggestion
            $data['sla_waiting_duration'] = $this->fields['sla_waiting_duration'] ?: 0;
```
`??` isn't going to work when we force every field to be an empty string by default (see `CommonDBTM::getEmpty`).

Reply:
I guess this bug was caused by #16383, so this should bring functionality back to what it was before.

Reply:
This would probably be better, to ensure that any unexpected value will be cast to an integer.
Casting the value in `$data` is important, as this value is copied into the item `$input` and can result in a SQL error if it is not an integer, see #16498.
```suggestion
            $data['sla_waiting_duration'] = (int) ($this->fields['sla_waiting_duration'] ?: 0);
```

Reply:
This change was required too, in order to fix #16498.